elements/lisk-p2p/src/peer.ts

```diff
@@ -391,7 +392,13 @@ export class Peer extends EventEmitter {
 					},
 					(err: Error | undefined, responseData: unknown) => {
 						if (err) {
-							reject(err);
+							reject(
```

Comment:
this makes the `test/mocha/functional/ws/transport/blocks.js` to fail because this `err` was not a `Error`, but it seems it's a string (ie err.message doesn't exist)